dashboards-observability/package.json

```diff
@@ -17,8 +17,11 @@
     "@nteract/outputs": "^3.0.11",
     "@nteract/presentational-components": "^3.4.3",
     "@reduxjs/toolkit": "^1.6.1",
+    "ag-grid-community": "^27.3.0",
+    "ag-grid-react": "^27.3.0",
```

Comment:
Why do we need to add this new dependency? Is there an issue where we discussed what was not available in EUI components. Adding more packages, puts lot more maintenance issues in long term.  Thus, we should have strong reasons to support addition of this package.

Reply:
hi @ps48  ,
We have changed table view implementation as per [https://github.com/opensearch-project/observability/issues/772](url).And as per new implementation, new dependencies are added and we have confirmed it on call and tkt too.

thanks,
Amruta